proco/schools/api.py

```diff
@@ -4,13 +4,14 @@
 from django.views.decorators.cache import cache_page
 
 from rest_framework import mixins, viewsets
+from rest_framework.generics import ListAPIView
 
 from django_filters.rest_framework import DjangoFilterBackend
 
 from proco.connection_statistics.models import SchoolWeeklyStatus
 from proco.locations.models import Country
 from proco.schools.models import School
-from proco.schools.serializers import ListSchoolSerializer, SchoolSerializer
+from proco.schools.serializers import BaseSchoolSerializer, ListSchoolSerializer, SchoolSerializer
```

Comment:
let's return **only** geopoints. response for such number of objects should be very big and at least half of it - unneeded information (name)